nucypher/blockchain/eth/deployers.py

```diff
@@ -587,32 +636,60 @@ def deploy(self,
         if gas_limit:
             origin_args.update({'gas': gas_limit})  # TODO: Gas Management - #842
 
-        # 1 - Deploy #
-        if emitter:
-            emitter.message(f"\nNext Transaction: {self.contract_name} Contract Creation", color='blue', bold=True)
-        the_escrow_contract, deploy_receipt = self._deploy_essential(contract_version=contract_version,
-                                                                     gas_limit=gas_limit,
-                                                                     confirmations=confirmations,
-                                                                     **overrides)
+        if deployment_mode is INIT:
+            # 1 - Deploy Stub
+            if emitter:
```

Comment:
This console logging can be moved above the conditional since it's the same in both cases.

Reply:
✔️